6-pubsub/package.json

```diff
@@ -8,8 +8,9 @@
     "start": "node ${SCRIPT:-app.js}",
     "monitor": "nodemon ${SCRIPT:-app.js}",
     "deploy": "gcloud preview app deploy app.yaml worker.yaml",
-    "lint": "jshint --exclude-path=.gitignore .",
-    "test": "npm run lint",
+    "lint": "jshint --exclude-path=../.gitignore .",
+    "mocha": "mocha -R dot test/unit/*.test.js",
```

Comment:
will this run mocha from the installed dev dependency?  or does it still require npm install -g mocha?